readthedocs/vcs_support/backends/git.py

```diff
@@ -310,7 +310,7 @@ def lsremote(self, include_tags=True, include_branches=True):
         if include_branches:
             extra_args.append("--heads")
 
-        cmd = ["git", "ls-remote", *extra_args, self.repo_url]
+        cmd = ["git", "ls-remote", *extra_args, "--", self.repo_url]
```

Comment:
This can get the default branch without cloning!

`git ls-remote --symref -q https://github.com/readthedocs/readthedocs.org HEAD`

Reply:
Went with a different approach, so we don't need this.